.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
from the top directory of your project, run the following:

```bash
git rm .vscode/lauch.json
git commit -m 'vscode cleanup'
git push
```

This will remove this file from the pull request (adding it to the gitignore is just the first step)

Reply:
this will resolve the second "open conversation"

Reply:
it still runs after having removed the files! So I'll delete the .vscode_temp now, too.